lib/l10n/intl_ro.arb

```diff
@@ -45,11 +45,14 @@
   "labelEven": "Pară",
   "labelOdd": "Impară",
   "labelDay": "Zi",
+  "labelToday": "Astăzi",
+  "labelTomorrow": "Mâine",
+  "labelNow": "Acum",
 
   "sectionShortcuts": "Scurtături",
   "sectionEvents": "Evenimente",
   "sectionFrequentlyAccessedWebsites": "Website-uri favorite",
-  "sectionEventsComingUp": "Evenimente următoare",
+  "sectionEventsComingUp": "Evenimente viitoare",
```

Comment:
Why viitoare if it includes ongoing events? I feel like următoare is a better word here

Reply:
Overlaps with "Arata mai multe" and throws an ugly render error